### ISSUE TITLE
RegEx to find package ID does not work for multiple domains in a single package

### ISSUE THREAD

**Issue by vwdss** [FlixMa/certbot-dns-strato#3]
RegEx to search for package ID does not work in my sitiation.

See following image of my Strato UI:

<img width="1173" alt="image" src="https://github.com/FlixMa/certbot-dns-strato/assets/3357905/4136295c-d148-4b1b-a5a6-06f7ac08d8dc">

There are two domains under one package. And in HTML I see `data-sortValue="xxx.me"`. However if I need to update second doman (`xxx.online`), it will never find it. 

HTML for this kind of UI looks like this:

```
<td data-label="Auftragsnummer">1234567 </td>
<td data-label="Kennung" data-sortValue="xxx.me">
    <div id="package_information_1234567" class="package-information">
        <p>xxx.me </p>
        <p>xxx.online </p>
        <p class="toggle mb-2 text-primary">
            <span class="show-more">
                <i class="fa fa-caret-right"></i>
                alle anzeigen
            </span>
            <span class="show-less">
                <i class="fa fa-caret-up"></i>
                weniger anzeigen
            </span>
        </p>
    </div>
</td>
<td>
    <div class="btn-group" role="group">
        <div class="dropdown">
            <button type="button" class="btn btn-default dropdown-toggle" data-bs-toggle="dropdown" type="button" aria-expanded="false">
                <i class="fa fa-ellipsis-h"></i>
            </button>
            <label id="dLabel" class="d-none">Dropdown fuer Aktionen </label>
            <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="dLabel">
                <li>
                    <a class="dropdown-item" href="/apps/CustomerService?sessionID=abc123&amp;cID=1&amp;node=kds_Vertragsbetreuung_2">
                        <i class="fa fa-cubes"></i>
                        Ihr Paket
                    </a>
                </li>
                <li>
                    <a class="dropdown-item" href="/apps/CustomerService?sessionID=abc123&amp;cID=1&amp;node=DomainOverview">
                        <i class="fa fa-globe"></i>
                        Domains verwalten
                    </a>
                </li>
                <li>
                    <a class="dropdown-item" href="/apps/CustomerService?sessionID=abc123&amp;cID=1&amp;node=EMailManagement">
                        <i class="fa fa-envelope"></i>
                        E-Mails verwalten
                    </a>
                </li>
                <li>
                    <a class="dropdown-item" href="/apps/CustomerService?sessionID=abc123&amp;cID=1&amp;node=DomainSearch">
                        <i class="fa fa-plus"></i>
                        Domains bestellen
                    </a>
                </li>
                <li>
                    <a class="dropdown-item" href="/apps/CustomerService?sessionID=abc123&amp;cID=1&amp;node=PackageChange&amp;type=upgrade">
                        <i class="fa fa-arrow-up"></i>
                        Paket upgraden
                    </a>
                </li>
                <li>
                    <a class="dropdown-item piwik_eventtracker" data-piwik-category="cancelation_dropdown_package" data-piwik-action="CustomerEntryPage" href="/apps/CustomerService?sessionID=abc123&amp;cID=0&amp;node=Cancelation&action_select_package=1234567">
                        <i class="fa fa-file-text"></i>
                        Paket kündigen
                    </a>
                </li>
            </ul>
        </div>
    </div>
</td>
```

**Comment by FlixMa** [comment]
Okay, thanks for mentioning this. 
The `cID` is 1 for both domains, right?

Could you maybe have a look at the page for setting txt-records and post the source of the relevant bits?
Are there any clues on how they are distinguished if it’s not via `cID`?

**Comment by FlixMa** [comment]
I dug a little further into the structure of strato packages.
It seems as `cID` is the identifier of the _package_, not the _domain_ (as I do have two domains listed, but each in their own package that wasn't too obvious to me).
The page for changing txt-records only depends on the `cID` and the `vhost` i.e. the domain name: /apps/CustomerService?sessionID=xxx&**cID=1**&node=ManageDomains&action_show_txt_records&**vhost=zzz.de**
As the web scraping part is the most vulnerable to UI changes anyway, it might be a good idea to just let the user feed the correct `cID` into this tool.
What do you think?

**Comment by vwdss** [comment]
@FlixMa Yeah, I think just giving a user a way to provide `cid` would be easier and less succeptible to bugs.

**Comment by PLanB2008** [comment]
Isn't this already fixed in the project this project is based on?

https://github.com/FlixMa/certbot-dns-strato/blob/3148d6dc696983cdea0f36ff5792cb3a87db9c8f/certbot_dns_strato/dns_strato.py#L220

**Comment by FlixMa** [comment]
Yes it seems so. Thanks!

https://github.com/Buxdehuda/strato-certbot/blob/dd640807a11267360c0df6d8105c90701e6db796/certbotstratoapi.py#L140

**Comment by FlixMa** [comment]
I just published version 0.2.2 on pypi. Please have a look if this fixes the issue and report back :-)
